platform/mbed_retarget.h

```diff
@@ -586,6 +592,12 @@ extern "C" {
     long telldir(DIR *);
     void seekdir(DIR *, long);
     int mkdir(const char *name, mode_t n);
+
+#if MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY
+    ssize_t minimal_console_write(const void *buffer, size_t length);
+    ssize_t minimal_console_read(void *buffer, size_t length);
+#endif // MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY
```

Comment:
We should explain that this is to replace the mbed_override_console functionality.
Should this be declared in `mbed` namespace` instead of here?

Reply:
[Done](https://github.com/hugueskamba/mbed-os/compare/6d43312662ac5c445423925ad5c85915f0580201..686fe66feaa50c040384b25636e6a7906a59779d).

Reply:
I don't see the update. 
Also you need to add doxygen function headers. I suggest to move this declaration in an `else` clause after the mbed_override_console functions.